libs/utils/src/environment_variable.ts

```diff
@@ -29,6 +29,8 @@ export enum BooleanEnvironmentVariableName {
   // Skips election hash checks when scanning to allow using old fixtures in
   // development even as their respective election definitions change.
   SKIP_SCAN_ELECTION_HASH_CHECK = 'REACT_APP_VX_SKIP_SCAN_ELECTION_HASH_CHECK',
+  // Skips authentication (i.e. signature verification) of ballot packages on import
+  SKIP_BALLOT_PACKAGE_AUTHENTICATION = 'REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION',
```

Comment:
Realized that it could be nice to skip this validation in dev, where we do manually modify ballot package files